src/popup/components/Truncate.vue

```diff
@@ -4,6 +4,7 @@
     :class="{ right }"
   >
     <span
+      ref="container"
```

Comment:
Does `Truncate` change relly blong to this commit?

Reply:
No, but after the changes in the App and Header components there were issues with rendering this component caused by the lack of the `parentElement` so I needed to fix this within this PR.